aries-site/src/pages/components/layer/side-drawer-layer.mdx

```diff
@@ -0,0 +1,123 @@
+
+### When to use double confirmation
+
+When data has been edited or changed and the user tries to close the layer, a double confirmation should be shown. This helps avoid unintentional loss of work.
+
+Learn more about the anatomy and implementation of a [double confirmation](/components/layer#when-to-use-double-confirmation) on the main layer page since this behavior applies to all layer types.
+
+## Scroll behavior
+
+The default behavior of the layer is to allow all of the content to scroll, but this behavior can be modified depending on the layer's contents.
+
+| Layer contents            | Sticky region                                                                                         | Scroll region   | Notes                                                                                                                            |
+| ------------------------- | ----------------------------------------------------------------------------------------------------- | --------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| **Filters**               | Footer                                                                                                | Header and body | A sticky footer ensures the user always has access to apply, reset, or cancel out of the filter view and return to the data set. |
+| **Form**                  | None or header can be sticky/appear on scroll up if this added context is valueable for the use case. | Body and footer | The footer actions should be at the bottom of the form to encourage the user to see the entire form before submission.           |
+| **Informational content** | TBD                                                                                                   | TBD             | TBD                                                                                                                              |
+
+### Sticky header
+
+If it would be valuable for the user to maintain the context of the layer's topic, the header can be made persistent or reappear when the user scrolls up.
+
+#### Persistent
+
+If it's necessary for the user to always be able to see the layer heading, make the heading persistent and sticky.
+
+<Example
+  docs="https://v2.grommet.io/layer?theme=hpe#props"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/LayerStickyExample.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/MonitorFormExample.js',
+  ]}
+  height={{ min: 'small' }}
+>
+  <LayerStickyExample />
+</Example>
+
+#### Scroll up
```

Comment:
Same comment as the "Persistent" example. Also, would this example benefit from having more content? Confused as to whether the header only appears once the scroll region reaches the header, or if the header appears whenever the user scrolls up regardless of whether or not it's near the header.

Reply:
I'm inclined not to add more content because then the form might be more suited for a fullscreen layer. The header will appear whenever the users scrolls up, not just when they're near the top.

Let's leave as is and see if we get questions (this is an existing example and we haven't been getting any questions so far).